errors/errors.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018 The Decred developers
+// Copyright (c) 2018-2019 The Decred developers
```

Comment:
this file hasn't been edited this year, not sure why you are bumping the copyright. please check through all of these date bumps in this PR and make sure none of them are unnecessary.

Reply:
doh, you fixed a spelling mistake too. that's fine.